instagram/src/authentication/withAuthenticate.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const withAuthenticate = FirstComponent => SecondComponent => props => 
```

Comment:
```suggestion
const withAuthenticate = FirstComponent => SecondComponent =>
```
You probably don't need props so try without it